src/crimson/os/seastore/omap_manager/btree/omap_btree_node_impl.cc

```diff
@@ -69,16 +69,17 @@ OMapInnerNode::handle_split_ret
 OMapInnerNode::handle_split(omap_context_t oc, internal_iterator_t iter,
                                mutation_result_t mresult)
 {
-  logger().debug("{}: {}","OMapInnerNode",  __func__);
+  logger().debug("OMapInnerNode: {}",  __func__);
   if (!is_pending()) {
     auto mut = oc.tm.get_mutable_extent(oc.t, this)->cast<OMapInnerNode>();
     auto mut_iter = mut->iter_idx(iter.get_index());
     return mut->handle_split(oc, mut_iter, mresult);
   }
   auto [left, right, pivot] = *(mresult.split_tuple);
-  //update will not cause overflow do it first.
+  //update operation will not cause node overflow, so we can do it first.
   journal_inner_update(iter, left->get_laddr(), maybe_get_delta_buffer());
-  if (!extent_will_overflow(pivot.size() + 1, std::nullopt)) {
+  bool overflow = extent_will_overflow(pivot.size() + 1, std::nullopt);
+  if (!overflow) {
```

Comment:
Why introduce a variable here?

Reply:
Ronen thought it is too long to put it in if condition express.